src/Ryujinx.HLE/HOS/Services/Audio/IHardwareOpusDecoderManager.cs

```diff
@@ -154,6 +154,28 @@ public ResultCode GetWorkBufferSizeForMultiStreamEx(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [CommandCmif(8)] // 16.0.0+
+        // GetWorkBufferSizeExEx(OpusParametersEx) -> u32
+        public ResultCode GetWorkBufferSizeExEx(ServiceCtx context)
+        {
+            // NOTE: GetWorkBufferSizeEx use hardcoded values to compute the returned size.
+            //       GetWorkBufferSizeExEx fixes that by using dynamic values.
+            //       Since we already doing that, it's fine the call it directly.
```

Comment:
```suggestion
            //       Since we're already doing that, it's fine to call it directly.
```
Same for the function below that has the exact same comment.